index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="style.css">
+    <title>Mo&apos;athal&apos;s Portfolio</title>
+</head>
+
+<body>
+    <div class="container flex column">
+        <header class="toolbar flex">
+            <a class="logo" href="_blank"> Mo&apos;athal S. Kachi</a>
+            <nav class="button"></nav>
+        </header>
+
+        <section class="headline flex column">
+            <div class="head flex column">
+                <h1 class="mainTxt">Mo&apos;athal</h1>
+
+                <h3 class="spprtTxt">Software Developer</h3>
+            </div>
+
+            <div class="spprtSect flex column">
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+
+                <hr class="divider">
+
+                <div class="sect flex">
+                    <div class="frame">
+                        <h4>Languages</h4>
+                        <span>SQL</span>
+                        <span>HTML</span>
+                        <span>CSS</span>
+                        <span>Javascript</span>
+                        <span>Ruby</span>
+                        <span>SQL</span>
+                    </div>
+
+                    <div class="frame">
+                        <h4>Frameworks</h4>
+                        <span>Flask</span>
+                        <span>Ruby on Rails</span>
+                        <span>SQlite</span>
+                        <span>Bootstrap</span>
+                    </div>
+
+                    <div class="frame">
+                        <h4>Skills</h4>
+                        <span>Codekit</span>
+                        <span>Github</span>
+                        <span>Gitlab</span>
+                        <span>Codepen</span>
+                        <span>Terminal</span>
+                    </div>
+                </div>
+
```

Comment:
- [ ] @Moathal great work implementing the changes requested, however, there is a small mistake on your languages and frameworks, ensure the content is aligned as in the given design:
<img width="178" alt="image" src="https://user-images.githubusercontent.com/61408860/176714925-2f8928b0-f713-403b-8224-f732ab28240b.png">